client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -120,6 +131,46 @@ class SiteImporterInputPane extends React.Component {
 		}
 	};
 
+	fetchEndpoints = () => {
+		wpcom.wpcom.req
+			.get( {
+				path: `/sites/${ this.props.site.ID }/site-importer/list-endpoints`,
+				apiNamespace: 'wpcom/v2',
+			} )
+			.then( resp => {
+				const twoWeeksAgo = moment().subtract( 2, 'weeks' );
```

Comment:
Is there some kind of limit on the number of items returned from `/site-importer/list-endpoints`? I'm just wondering if the response will start to get a bit big over time.

Reply:
Limit is 50 on the backend so I think we are fine doing the filtering client-side. We can always move it to backend if needed.

Reply:
Lovely, thanks Marek!